Validation/HGCalValidation/scripts/hgcalPerformanceValidation.py

```diff
@@ -22,6 +22,10 @@
 from Validation.RecoTrack.plotting.validation import Sample, Validation
 from Validation.HGCalValidation.hgcalHtml import _sampleName,_pageNameMap,_summary,_summobj,_MatBudSections,_geoPageNameMap,_individualmaterials,_matPageNameMap,_individualmatplots,_individualMatPlotsDesc,_hideShowFun,_allmaterialsplots,_allmaterialsPlotsDesc, _fromvertexplots, _fromVertexPlotsDesc
 
+from RecoHGCal.TICL.iterativeTICL_cff import ticlIterLabelsMerge
+trackstersIters = ['ticlTracksters'+iteration for iteration in ticlIterLabelsMerge]
+trackstersIters.extend(["ticlSimTracksters", "ticlSimTracksters_fromCPs"])
```

Comment:
Since this list is already extended in `makeHGCalValidationPlots.py` and that is the command the list is used for, I would avoid code duplication here and just `import trackstersIters from Validation/HGCalValidation/scripts/makeHGCalValidationPlots`

Reply:
It cannot be imported because we keep this in scripts and not in python folder so that we can run it also as an executable.

Reply:
OK. Then we should probably define it in the `PostProcessorHGCAL_cfi` (where the same extension is again executed) and import from there.
It can be done in the next PR.